docs/getting_started/tutorial.md

```diff
@@ -243,17 +252,16 @@ class BlogIndexPage(Page):
     ]
 ```
 
-Run `python manage.py makemigrations` and `python manage.py migrate`.
-
-Since the model is called `BlogIndexPage`, the default template name
-(unless we override it) will be `blog_index_page.html`. Django will look for a template whose name matches the name of your Page model within the templates directory in your blog app folder. This default behaviour can be overridden if needed.  
-To create a template for the `BlogIndexPage` model, create a file at the location `blog/templates/blog/blog_index_page.html`.
+Since you have made changes to `model.py` run the migrations:
 
-```{note}
-You may need to create the folders `templates/blog` within your `blog` app folder
+```console
+$ python manage.py makemigrations
+$ python manage.py migrate
 ```
 
-In your `blog_index_page.html` file enter the following content:
+Since the model is called `BlogIndexPage`, the default template name
+(unless we override it) will be `blog/templates/blog/blog_index_page.html`. Create this file
+with the following content:
```

Comment:
This isn’t as good as what we’re replacing. The template name is just` blog_index_page.html`, not the full path, and information on where Django looks for it is valuable.